### ISSUE TITLE
PPA Launchpad versioning is broken

### ISSUE THREAD

**Issue by Donokami** [numixproject/numix-icon-theme-circle#143]
Hi,
The last versions of Numix Circle are not pushed on Ubuntu, because the versioning seems broken. When forcing to install the `19.12.27+202111201922~ubuntu21.04.1`, we got this warning: 
```
pkg: warning: downgrading numix-icon-theme-circle from 20.09.19-1 to 19.12.27+202111201922~ubuntu21.04.1
```
Even if the last version is not `20.09.19-1`.

**Comment by Foggalong** [comment]
![Screenshot from 2021-11-30 14-07-31](https://user-images.githubusercontent.com/5920259/144062126-1ecefd9a-2ef2-4a0b-80cc-f9ad1b0f09b0.png)

This will be with `19.12.27` being the start of the version string for the 21.04 PPA package while `20.09.19` is the start of the string in the distribution package, so it compares them an says "oh this is less than that" even though for the PPA it's the second part of the string which defines the version. My guess is 20.10 and 21.10 are both having this problem too

Any ideas @khurshid-alam?

**Comment by khurshid-alam** [comment]
@Donokami 

Please use the repo...you don't need ppa.

It is picking package version from tag...let me check.

**Comment by ocobblepot** [comment]
Bumping this as there is still a problem with the versioning/package naming for numix-icon-theme-circle in the PPA. On Ubuntu 22.04 Jammy, apt picks the version in the Ubuntu repos (22.01.10-1) which does not include recent commits - for example, [https://github.com/numixproject/numix-core/commit/b13f030e95d9ebbbbb3561c2fe8d864e624db752](url) and [https://github.com/numixproject/numix-core/commit/a0143d2f10cbd34dce72b0cc8aa1e3cc8193601d](url). These commits are included in the version in the PPA - 19.12.27+202210171917~ubuntu22.04.1.

Numix Square is unaffected - apt selects the 22.10.17+202210171902~ubuntu22.04.1 version in the PPA.

I can work around this by specifying `apt install numix-icon-theme-circle=19.12.27+202210171917~ubuntu22.04.1` and then `apt-mark hold numix-icon-theme-circle`, but given that the package naming for numix-icon-theme-square is correct, a fix for numix-icon-theme-circle in the PPA is needed.

**Comment by ocobblepot** [comment]
This issue is still present on Launchpad @khurshid-alam 

With the Numix PPA added to repositories, the correct version 22.11.26+202211261516 of Numix Square is installed, but the misnamed version 19.12.27+202211261546 of Numix Circle is ignored in favour of the older version in the Ubuntu repo.

[https://launchpad.net/~numix/+archive/ubuntu/ppa?field.series_filter=jammy](url)

**Comment by ocobblepot** [comment]
> Numix PPA required new debian packaging if installing from PPA.

Sorry, I'm unclear about this response.

If the Numix Circle packages in the Numix PPA are given the same name format as the Numix Square packages, then the problem will be resolved.

**Comment by ocobblepot** [comment]
... and the problem persists with the 23.01.02 release. Numix Square updates fine from the PPA, the incorrectly named updated package for Numix Circle is ignored :(

**Comment by khurshid-alam** [comment]
Yes. It is taking the version manually and not from got tag. I will make another rebuild.

**Comment by ocobblepot** [comment]
... and the problem persists again with the latest Launchpad uploads yesterday https://launchpad.net/~numix/+archive/ubuntu/ppa?field.series_filter=jammy

**Comment by khurshid-alam** [comment]
I fixed that last week. But it broke again with Jammy above series. Perhaps I can copy the debian folder from debian. But it will most likely break older versions.

**Comment by msangel** [comment]
It does not have Release file for Ubuntu 26.04 (Resolute)
